### PR TITLE
Improve code quality on members list RHS

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
Improves the code quality of the channel members RHS list by address nits detected by @julmondragon on my previous PR

@stevemudie there's no changes in features here or bugfix just code improvement, so I guess QA is making sure it works as before

#### Ticket Link
N/A

#### Related Pull Requests
N/A

#### Screenshots
N/A

#### Release Note

```release-note
NONE
```

### CODE REVIEW

**Review by julmondragon — APPROVED** [review]
LGTM! Thanks

File: components/channel_members_rhs/channel_members_rhs.tsx [hunk 1]
```diff
@@ -250,7 +258,7 @@ export default function ChannelMembersRHS({
     );
 }
 
-const MemberListSerparator = styled.div`
+const MemberListSeparator = styled.div`
```
Comment (pvev) [inline components/channel_members_rhs/channel_members_rhs.tsx:261]: 👍

**Review by pvev — APPROVED** [review]
LGTM! thanks @JulienTant

**Review by stevemudie — APPROVED** [review]
Worked as expected. This is all set @JulienTant

**Pull request merged**